src/Apps/Juno/Client.hs

```diff
@@ -121,92 +122,18 @@ snapServer :: InChan (RequestId, [CommandEntry]) -> CommandMVarMap -> IO ()
 snapServer toCommands cmdStatusMap = httpServe serverConf $
     applyCORS defaultOptions $ methods [GET, POST]
     (ifTop (writeBS "use /hopper for commands") <|>
-     route [ ("/api/juno/v1/accounts/create", createAccounts toCommands cmdStatusMap)
```

Comment:
Move all "/api/juno/v1" into ApiHandler.hs, thus a single entry here:
```
("/api/juno/v1", apiHandler)
```
and then in ApiHandler.hs,
```
apiHandler :: ... -> Snap ()
apiHandler = route [("/accounts/create", createAccounts ...),("/accounts/adjust", adjustAccounts ...)]